cx_Freeze/hooks.py

```diff
@@ -816,12 +816,23 @@ def load_clr(finder, module):
     module_dir = os.path.dirname(module.file)
     dllname = 'Python.Runtime.dll'
     finder.IncludeFiles(os.path.join(module_dir, dllname), dllname)
-    
-    
+
+
+def _get_base_prefix():
+    """Get prefix to site-specific directory prefix where the platform
+    independent Python files are installed."""
+    try:
+        return sys.base_prefix  # Python 3.3+
+    except AttributeError:
+        # For Python 2, use sys.real_prefix set by virtualenv if set,
+        # otherwise fall back to sys.prefix
+        return getattr(sys, "real_prefix", "prefix")
```

Comment:
This code doesn't do what I think you want it to do -- namely, use sys.real_prefix unless it cannot be found, in which case return sys.prefix. Perhaps something like this instead?

return getattr(sys, "real_prefix", sys.prefix)